admin/resources/js/stories/Users.stories.tsx

```diff
@@ -1,8 +1,40 @@
 import React from "react";
 import { storiesOf } from "@storybook/react";
+import { createClient } from "urql";
 import UserTable from "../components/UserTable";
 import fakeUsers from "../fakeData/fakeUsers";
+import { AllUsersQuery, useAllUsersQuery } from "../api/generated";
+import ClientProvider from "../components/ClientProvider";
 
-storiesOf("Users", module).add("User Table", () => (
-  <UserTable users={fakeUsers} />
+const userData = fakeUsers as unknown as AllUsersQuery["users"];
```

Comment:
Instead of doing this awkward conversion. I set the fake users type directly in the file. Pushing now